protocols/Mass_Spec_Prep_2017-12-4/README.md

```diff
@@ -0,0 +1,62 @@
+# Mass Spectrometry Sample Prep
+
+### Author
+[Opentrons](http://www.opentrons.com/)
+
+### Partner
+
+
+## Categories
+* Proteins & Proteomics
+  * Sample Prep
+
+
+## Description
```

Comment:
There needs to be enough info here so that the user understands what format the CSVs need to be in, and which steps use the CSVs